### PR TITLE
DRAFT minutes for 2024-04-10

### PR DESCRIPTION
**Pull request by maximilien**
Added DRAFT minutes for review and approvals

### CODE REVIEW

**Review by ghost — APPROVED** [review]
thanks @maximilien! \o/

**Comment by maximilien** [comment]
Any chance for approval @ashman-p or @thb-sb or @ryjones so I can merge.

Thanks

File: meeting-minutes/2024/2024-04-10-TAC.md [hunk 1]
```diff
@@ -41,6 +96,16 @@ nav_exclude: true
...
+
+* Bryan (Keyfactor)
+* Norm ()
+* Ry (LF)
+* Hart (LF)
+* Naomi (LF)
+* Nigel (IBM)
+
+## Voting members
 * [ ] ~~Norman Ashley, Cisco~~
```
Comment (ashman-p) [inline meeting-minutes/2024/2024-04-10-TAC.md]:
```suggestion
* [x] ~~Norman Ashley, Cisco~~
```

File: meeting-minutes/2024/2024-04-10-TAC.md [hunk 2]
```diff
@@ -41,6 +96,16 @@ nav_exclude: true
 
 # Attended by
 
+## Non-voting members
+
+* Bryan (Keyfactor)
+* Norm ()
```
Comment (ashman-p) [inline meeting-minutes/2024/2024-04-10-TAC.md]:
```suggestion

```

**Review by ashman-p — CHANGES_REQUESTED** [review]

**Review by ashman-p — APPROVED** [review]

**Comment by ryjones** [comment]
@maximilien I think you can merge now?

**Review by sam-stanwyck — APPROVED** [review]

**Comment by maximilien** [comment]
Interesting @ryjones I don't see a merge option (button). What gives?

**Comment by maximilien** [comment]
Thanks @ryjones

**Comment by ryjones** [comment]
@maximilien you have not accepted your invite to join the PQCA org, which is why you don't have a merge button
Screenshot 2024-04-30 at 13 11 56

**Pull request merged**